packages/react/src/experimental/Alert/index.tsx

```diff
@@ -0,0 +1,106 @@
+import { Button } from "@/components/Actions/Button"
+import { Icon } from "@/factorial-one"
+import { ExternalLink } from "@/icons/app"
+import { cn, focusRing } from "@/lib/utils"
+import { cva, type VariantProps } from "cva"
+import { AlertAvatar } from "../Information/Avatars/AlertAvatar"
+
+type AlertVariant = "info" | "warning" | "critical"
+
+const alertVariants = cva({
+  base: "flex w-full flex-col items-start justify-between gap-4 rounded-md px-3 py-3 text-f1-foreground ring-1 ring-inset ring-f1-border-secondary sm:flex-row sm:items-center sm:px-4",
+  variants: {
+    variant: {
+      info: "bg-f1-background-info",
+      warning: "bg-f1-background-warning",
+      critical: "bg-f1-background-critical",
+    },
+  },
+  defaultVariants: {
+    variant: "info",
+  },
+})
+
+const titleVariants = cva({
+  variants: {
+    variant: {
+      info: "text-f1-foreground-info",
+      warning: "text-f1-foreground-warning",
+      critical: "text-f1-foreground-critical",
+    },
+  },
+  defaultVariants: {
+    variant: "info",
+  },
+})
+
+const buttonVariants: Record<AlertVariant, "outline" | "promote" | "critical"> =
+  {
+    info: "outline",
+    warning: "promote",
+    critical: "outline",
+  }
+
+interface AlertProps extends VariantProps<typeof alertVariants> {
+  title: string
+  description: string
+  primaryAction: {
+    label: string
+    onClick?: () => void
+  }
+  secondaryAction: {
```

Comment:
We always have a link here, not an action. Let's call it `link`. 

I see it is marked as mandatory here, but in the code you always check whether or not it exists. It must be optional prop then